common/cmake/devices.cmake

```diff
@@ -1143,6 +1143,7 @@ function(ADD_FPGA_TARGET)
   #   NAME <name>
   #   [TOP <top>]
   #   BOARD <board>
+  #   [INCLUDES <include list>]
```

Comment:
This shouldn't be required, if you set `add_file( ... SCANNER_TYPE verilog)`

Reply:
SCANNER_TYPE won't work because the files are generated during the build, and scanning happens at CMake config time.  Is there any way for fusesoc to output a dependency graph for use in CMake?

Reply:
I think that this is not possible directly. However, `fusesoc` produces auxiliary YAML file with the configuration data.
The information about the files can be extracted from that file.

Reply:
I agree that the "SCANNER_TYPE verilog" won't work, but there is still not reason to add the "INCLUDES" flag.

Reply:
See https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1648#discussion_r483595644